### PR TITLE
Documentation changes

### PR DESCRIPTION
**Pull request by dythya**
fixed links
changed titles and project to be SimPy Classic

### CODE REVIEW

File: docs/source/Manuals/SManual/SManual.rst [hunk 1]
```diff
@@ -1044,16 +1044,16 @@ the ``gasstation``.)
 
 
 .. _`simpydownload`: https://github.com/SimPyClassic/SimPyClassic
-.. _`Python web-site` : http://www.Python.org
+.. _`Python web-site` : https://www.Python.org
```
Comment (rhysmdnz) [inline docs/source/Manuals/SManual/SManual.rst]: Missed the capital p

File: docs/source/Manuals/SManual/SManual.rst [hunk 2]
```diff
@@ -1044,16 +1044,16 @@ the ``gasstation``.)
...
 
 .. .. _Top: Contents_
 
 .. _monitors: `Monitors and Recording Simulation Results`_
 
 
-.. _`Python random module`: http://www.python.org/doc/current/lib/module-random.html
-.. _Python: http://www.Python.org
+.. _`Python random module`: https://docs.python.org/3/library/random.html
+.. _Python: https://www.Python.org
```
Comment (rhysmdnz) [inline docs/source/Manuals/SManual/SManual.rst]: Capital p again

**Review by rhysmdnz — CHANGES_REQUESTED** [review]

**Comment by rhysmdnz** [comment]
Also is there supposed to be a space in simpyclassic as there isn't one in the readme

File: docs/source/Manuals/SManual/SManual.rst (hunk 1 above)
Comment (dythya) [reply docs/source/Manuals/SManual/SManual.rst]: oops will fix

File: docs/source/Manuals/SManual/SManual.rst (hunk 2 above)
Comment (dythya) [reply docs/source/Manuals/SManual/SManual.rst]: :-( will grep.  I was relying on the link checker - it complained for some of them

**Comment by dythya** [comment]
Interesting sometimes you can reply.
I think it should be SimPy Classic when in a sentence and used for naming.  Otherwise I think it is hard
to read.  I will fix the readme

File: docs/source/Manuals/Examples/index.rst [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+SimPy Classic - Examples
 ==========================================
 
 These are the examples that ship with the SimPy source.
 
 These pages are to index the examples that ship with SimPy and give
 users another way to find them.
 
-.. _`Python web-site` : http://www.Python.org
+.. _`Python web-site` : http://www.yython.org
```
Comment (rhysmdnz) [inline docs/source/Manuals/Examples/index.rst]: that's a y

Comment (rhysmdnz) [reply docs/source/Manuals/Examples/index.rst]: also https

File: docs/source/Manuals/SManual/SManual.rst [hunk 4]
```diff
@@ -11,7 +11,7 @@ Introduction to SimPy
 
 :Authors: G A Vignaux and Klaus Muller
 :Date:  |today|
-:SimPy release: |release|
+:Release: |release|
```
Comment (rhysmdnz) [inline docs/source/Manuals/SManual/SManual.rst:14]: Some places say release some say SimPy Classic Release

**Review by rhysmdnz — CHANGES_REQUESTED** [review]

File: docs/source/Manuals/SManual/SManual.rst (hunk 1 above)
Comment (dythya) [reply docs/source/Manuals/SManual/SManual.rst]: fixed

File: docs/source/Manuals/SManual/SManual.rst (hunk 2 above)
Comment (dythya) [reply docs/source/Manuals/SManual/SManual.rst]: fixed

File: docs/source/Manuals/SManual/SManual.rst (hunk 4 above)
Comment (dythya) [reply docs/source/Manuals/SManual/SManual.rst:14]: Where I said that the manual title did not have Simpy Classic in it

Comment (dythya) [reply docs/source/Manuals/SManual/SManual.rst:14]: fixed any that needed it

**Review by rhysmdnz — APPROVED** [review]

**Pull request merged**